Framework/DataHandling/inc/MantidDataHandling/LoadEventNexus.h

```diff
@@ -7,14 +7,21 @@
 #include "MantidAPI/IFileLoader.h"
 #include "MantidDataHandling/BankPulseTimes.h"
 #include "MantidDataObjects/EventWorkspace.h"
-#include <nexus/NeXusFile.hpp>
-#include <nexus/NeXusException.hpp>
 #include "MantidDataObjects/Events.h"
 #include "MantidAPI/WorkspaceGroup.h"
 #include "MantidGeometry/Instrument.h"
 #include "MantidGeometry/Instrument/ParameterMap.h"
 #include "MantidKernel/TimeSeriesProperty.h"
 #include "MantidDataHandling/EventWorkspaceCollection.h"
+
+#ifdef _WIN32 // fixing windows issue causing conflict between
+// winnt char and nexus char
+#undef CHAR
```

Comment:
Where is `CHAR` defined - I've looked and it's not a standard define from the [MSVC compiler](https://msdn.microsoft.com/en-us/library/b0084kay.aspx) and grepping for `#define CHAR` isn't revealing anything? 
Although it's extremely unlikely if anyone relies on the previous definition of `CHAR` and Nexus `CHAR` redefines it by doing this it could lead to issues.